solid-terms.ttl

```diff
@@ -1,212 +1,283 @@
     rdfs:label "registry class"@en .
 
-:inbox
-    a rdf:Property ;
+solid:inbox
     owl:deprecated true ;
-    owl:equivalentProperty ldp:inbox;
-    rdfs:subPropertyOf ldp:inbox;
+    a rdf:Property ;
+    dc:issued "2015-11-16"^^xsd:date ;
+    dc:modified "2016-02-05"^^xsd:date ;
+    dc:modified "2018-10-25"^^xsd:date ;
+    owl:equivalentProperty ldp:inbox ;
+    rdfs:subPropertyOf ldp:inbox ;
     rdfs:comment "Deprecated pointer to a Linked Data Notifications inbox; please use http://www.w3.org/ns/ldp#inbox instead."@en ;
-    rdfs:isDefinedBy <> ;
+    rdfs:isDefinedBy <http://www.w3.org/ns/solid/terms> ;
     rdfs:label "inbox (deprecated)"@en .
 
-:inserts
+solid:inserts
     a rdf:Property, owl:ObjectProperty, owl:FunctionalProperty ;
+    dc:issued "2017-08-17"^^xsd:date ;
     rdfs:comment "The triple patterns this patch adds to the document."@en ;
-    rdfs:isDefinedBy <> ;
+    rdfs:isDefinedBy <http://www.w3.org/ns/solid/terms> ;
     rdfs:label "inserts"@en ;
-    rdfs:domain :Patch ;
+    rdfs:domain solid:Patch ;
     rdfs:range log:Formula .
 
-:instance
+solid:instance
     a rdf:Property ;
+    dc:issued "2018-01-24"^^xsd:date ;
     rdfs:comment "Maps a type to an individual resource, typically an index or a directory listing resource."@en ;
-    rdfs:isDefinedBy <> ;
+    rdfs:isDefinedBy <http://www.w3.org/ns/solid/terms> ;
     rdfs:label "instance"@en .
 
-:instanceContainer
+solid:instanceContainer
     a rdf:Property ;
+    dc:issued "2018-01-24"^^xsd:date ;
     rdfs:comment "Maps a type to a container which the client would have to list to get the instances of that type."@en ;
-    rdfs:isDefinedBy <> ;
+    rdfs:isDefinedBy <http://www.w3.org/ns/solid/terms> ;
     rdfs:label "instance container"@en .
 
-:loginEndpoint
+solid:loginEndpoint
     a rdf:Property ;
+    dc:issued "2015-11-16"^^xsd:date ;
     rdfs:comment "The login URI of a given server."@en ;
-    rdfs:isDefinedBy <> ;
+    rdfs:isDefinedBy <http://www.w3.org/ns/solid/terms> ;
     rdfs:label "loginEndpoint"@en .
 
-:logoutEndpoint
+solid:logoutEndpoint
     a rdf:Property ;
+    dc:issued "2015-11-16"^^xsd:date ;
     rdfs:comment "The logout URI of a given server."@en ;
-    rdfs:isDefinedBy <> ;
+    rdfs:isDefinedBy <http://www.w3.org/ns/solid/terms> ;
     rdfs:label "logoutEndpoint"@en .
 
-:notification
+solid:notification
     a rdf:Property ;
+    dc:issued "2016-03-11"^^xsd:date ;
+    dc:modified "2018-10-25"^^xsd:date ;
     rdfs:comment "Notification resource for an inbox."@en ;
-    rdfs:isDefinedBy <> ;
+    rdfs:isDefinedBy <http://www.w3.org/ns/solid/terms> ;
     rdfs:label "notification"@en .
 
-:privateTypeIndex
+solid:oidcIssuer
     a rdf:Property ;
+    dc:issued "2017-08-15"^^xsd:date ;
+    dc:modified "2017-08-16"^^xsd:date ;
+    rdfs:comment "The preferred OpenID Connect issuer URI for a given Web ID."@en ;
+    rdfs:isDefinedBy <http://www.w3.org/ns/solid/terms> ;
+    rdfs:label "oidc issuer"@en ;
```

Comment:
```suggestion
    rdfs:label "OIDC issuer"@en ;
```

Can't remember at the moment what the practice is for property labels starting with abbr.. Feel free to consider "OIDC".. look it up somewhere?

Reply:
It's a bit all over the place:
- Capitalisation of all words
- Capitalisation of the first word
- Replica of the URI minus namespace 
- Keep everything lowercase

I feel like the best practice should be the solution you came up with on this instance:
- Keep the canonical upper/lowercasing of acronyms.

I think additionally, I would consider Capitalising the first letter of the label. We shouldn't infer meaning from the casing used in labels. Therefore, the best practice should be in line with all literals. Keep Capitalisation rules consistent.